src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
According to the BEM methodology, modifiers should be used with the block or element name. For example, if 'calendar' is the block, 'calendar--start-day-sun' and 'calendar--month-length-31' should be used alongside the base class 'calendar'.